Makefile

```diff
@@ -1,32 +1,49 @@
 CRATE_NAME := x25519-chacha20poly1305
-
+SCOPE_NAME := entropyxyz
```

Comment:
Name of the npm namespace / scope for entropy org. https://docs.npmjs.com/cli/v9/using-npm/scope#publishing-scoped-packages